CanvasColorSpaceProposal.md

```diff
@@ -105,11 +122,15 @@ The pixelFormat attributes specifies the numeric types to be used for storing pi
 #### Selecting the best color space match for the user agent's display device
 <pre>
 var colorSpace = window.matchMedia("(color-gamut: rec2020)").matches ? "rec2020" : 
-    (window.matchMedia("(color-gamut: p3)").matches ? "p3" : "srgb");
+    ( window.matchMedia("(color-gamut: p3)").matches ? "p3" : 
+      ( window.matchMedia("(color-gamut: rec2100pq)").matches ? "rec2100pq" :
+        ( window.matchMedia("(color-gamut: rec2100pqn)").matches ? "rec2100pqn" : "srgb" )
+      )
+    );
 </pre>
 
 #### Selecting the best pixelFormat for the user agent's display device
-Selection should be based on the best color space match (see above). For srgb, at least 8 bits per component is recommended; for p3, 10 bits; and for rec2020, 12 bits.  The float16 format is suitable for any colorspace.  There may soon be a proposal to add a way of detecting HDR displays, possibly something like "window.screen.isHDR()" (TBD), which would be a good hint to use the float16 format.
+Selection should be based on the best color space match (see above). For srgb, at least 8 bits per component is recommended; for p3, 10 bits; and for rec2020 and rec2100pq and rec2100pqn, 12 bits.  The float16 format is suitable for any colorspace.  There may soon be a proposal to add a way of detecting HDR displays, possibly something like "window.screen.isHDR()" (TBD), which would be a good hint to use the float16 format.
```

Comment:
Media Queries 5 will allow querying both the screen and the video plane (in the cases where that is different, such as TVs) for color gamut and for dynamic range. See https://github.com/w3c/csswg-drafts/issues/4471#issuecomment-549178832